src/adapters/utils/networkUtils.js

```diff
@@ -143,8 +144,8 @@ const processAxiosResponse = (clientResponse) => {
     }
     // (edge case) response and code is not present
     return {
-      response: '',
-      status: 500,
+      response: clientResponse?.response?.data || '',
```

Comment:
data will be there normally for all?

Reply:
Usually, we have seen that, there is some data in response body(`response.data`)